drivers/pinmux/stm32/pinmux_stm32f3.h

```diff
@@ -26,12 +26,12 @@
 #endif
 
 #if CONFIG_SOC_STM32F303XC
+#define STM32F3_PINMUX_FUNC_PA9_I2C2_SCL    (STM32_PINMUX_ALT_FUNC_4 | STM32_OPENDRAIN_PULLUP)
+#define STM32F3_PINMUX_FUNC_PA10_I2C2_SDA   (STM32_PINMUX_ALT_FUNC_4 | STM32_OPENDRAIN_PULLUP)
 #define STM32F3_PINMUX_FUNC_PB6_I2C1_SCL    (STM32_PINMUX_ALT_FUNC_4 | STM32_OPENDRAIN_PULLUP)
 #define STM32F3_PINMUX_FUNC_PB7_I2C1_SDA    (STM32_PINMUX_ALT_FUNC_4 | STM32_OPENDRAIN_PULLUP)
 #define STM32F3_PINMUX_FUNC_PB10_USART3_TX  (STM32_PINMUX_ALT_FUNC_7 | STM32_PUSHPULL_PULLUP)
 #define STM32F3_PINMUX_FUNC_PB11_USART3_RX  (STM32_PINMUX_ALT_FUNC_7 | STM32_PUSHPULL_PULLUP)
-#define STM32F3_PINMUX_FUNC_PF0_I2C2_SDA    (STM32_PINMUX_ALT_FUNC_4 | STM32_OPENDRAIN_PULLUP)
```

Comment:
Removing these defines is not required.
They are still valid and could be useful for other boards

Reply:
Fixed